client-js/app/elements/labrad-registry.html

```diff
@@ -102,87 +102,74 @@
           <template is="dom-repeat" items="{{keys}}">
             <tr name="{{item.name}}">
               <td>
-                <iron-icon icon="more-vert"></iron-icon>
-              </td>
-              <td>
                 <dir-item kind="key">{{item.name}}</dir-item>
               </td>
               <td class="label-wide">
                 <form is="iron-form" id="{{item.name}}">
-                  <iron-a11y-keys keys="tab" on-keys-pressed="incrementSelector">
-                    <paper-input-container>
-                      <input is="iron-input" name="{{item.name}}" value="{{item.value}}">
-                    </paper-input-container>
-                  </iron-a11y-keys>
+                  <paper-input-container no-label-float>
+                    <input is="iron-input" name="{{item.name}}" value="{{item.value}}">
+                  </paper-input-container>
                 </form>
               </td>
             </tr>
           </template>
         </tbody>
-      </div>
-    </table>
+      </table>
+    </div>
+
+    <!-- Dialogs for various registry operations -->
 
-    <!-- These are the dialogs -->
-    <paper-dialog id="doRename">
-      <p>Rename "<span>{{selItem}}</span>" to something else</p>
+    <paper-dialog id="newKeyDialog" modal>
+      <h1>Create new key</h1>
       <form>
-        <input></input>
+        <paper-input id="newKeyInput" label="Key"></paper-input>
+        <paper-textarea id="newValueInput" label="Value"></paper-textarea>
```

Comment:
If I remember correctly the old delphi registry editor would basically supress or revert newlines it didn't like. Is this behavior we want to add here eventually?